src/classes/CW_AudienceCSVProcessHelperTest.cls

```diff
@@ -0,0 +1,145 @@
+		//==============================================================
+		CSVBatchReader batchJobRead = new CSVBatchReader(csvFile, CampRec.Id,sch.Id);
+    	Test.StartTest();
+      	Database.executeBatch(batchJobRead);
+  		Test.StopTest();
+	}
+	
+
+	static testMethod void ProcessBatchOnly()
+	{
+		//Nothibg to test just for build. Then we make rarl tests
+		CW_Campaign__c CampRec=TestDataUtil.getCampaign();
+		insert CampRec;
+		CW_CSV_Import_Status__c statRec = new CW_CSV_Import_Status__c(Marketing_Campaign__c= CampRec.ID);
+		insert statRec;
+		// Now import records - All a rejected s there no schedule/ rewards
+		//==============================================================
+		//CW_AudienceCSVProcessHelper ProcHelper= new CW_AudienceCSVProcessHelper(statRec.Id);
+		List<CW_CSV_Stage_AudienceMembers__c> csvLst=new List<CW_CSV_Stage_AudienceMembers__c>();
+		
+		csvLst.Add(new CW_CSV_Stage_AudienceMembers__c(Group_Name__c ='g1',PlayerId__c='p1',FreeBetId__c='fb1',CSV_Import_Status__c =statRec.Id ) );
+		csvLst.Add(new CW_CSV_Stage_AudienceMembers__c(Group_Name__c ='g1',PlayerId__c='p2',FreeBetId__c='fb1',CSV_Import_Status__c =statRec.Id ) );
+		csvLst.Add(new CW_CSV_Stage_AudienceMembers__c(Group_Name__c ='g2',PlayerId__c='p3',FreeBetId__c='fb1',CSV_Import_Status__c =statRec.Id ) );
+		csvLst.Add(new CW_CSV_Stage_AudienceMembers__c(Group_Name__c ='g2',PlayerId__c='p4',FreeBetId__c='fb1',CSV_Import_Status__c =statRec.Id ) );
+		csvLst.Add(new CW_CSV_Stage_AudienceMembers__c(Group_Name__c ='g3',PlayerId__c='p2',FreeBetId__c='fb1',CSV_Import_Status__c =statRec.Id ) );
+		csvLst.Add(new CW_CSV_Stage_AudienceMembers__c(Group_Name__c ='g3',PlayerId__c='p4',FreeBetId__c='fb1',CSV_Import_Status__c =statRec.Id ) );
+		// Rows that thont afect process ( duplicates )
+		csvLst.Add(new CW_CSV_Stage_AudienceMembers__c(Group_Name__c ='g1',PlayerId__c='p2',FreeBetId__c='fb1',CSV_Import_Status__c =statRec.Id ) );
+		csvLst.Add(new CW_CSV_Stage_AudienceMembers__c(Group_Name__c ='g2',PlayerId__c='p3',FreeBetId__c='fb1',CSV_Import_Status__c =statRec.Id ) );
+		csvLst.Add(new CW_CSV_Stage_AudienceMembers__c(Group_Name__c ='g2',PlayerId__c='p4',FreeBetId__c='fb1',CSV_Import_Status__c =statRec.Id ) );
+		// NO Free Bet no upload
+		
+		csvLst.Add(new CW_CSV_Stage_AudienceMembers__c(Group_Name__c ='g1',PlayerId__c='p3',FreeBetId__c='fb2',CSV_Import_Status__c =statRec.Id ) );
+		csvLst.Add(new CW_CSV_Stage_AudienceMembers__c(Group_Name__c ='g1',PlayerId__c='p4',FreeBetId__c='fb2',CSV_Import_Status__c =statRec.Id ) );
+		csvLst.Add(new CW_CSV_Stage_AudienceMembers__c(Group_Name__c ='g2',PlayerId__c='p1',FreeBetId__c='fb3',CSV_Import_Status__c =statRec.Id ) );
+		
+		insert csvLst;
+		
+		
+    	Test.StartTest();
+		CSVBatchProcess batchJobProcess = new CSVBatchProcess(statRec.ID,CSVBatchProcess.ProcesTypes.CreateAccounts);
+		Database.executeBatch(batchJobProcess);
+
+      	Test.StopTest();
+
+	}
+  
+  
+  	static testMethod void ProcessImport()
```

Comment:
* I suggest to annotate test methods with `@IsTest` for better readability.
* Methods should use the `lowerCamelCase` naming convention. I suggest to rename accordingly.